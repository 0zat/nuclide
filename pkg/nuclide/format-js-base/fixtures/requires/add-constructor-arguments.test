'use strict';

new one(two, {three});
