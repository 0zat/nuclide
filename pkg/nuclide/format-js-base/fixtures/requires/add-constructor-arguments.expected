'use strict';

var one = require('one');
var three = require('three');
var two = require('two');

new one(two, {three});
